fastestimator/pipeline.py

```diff
@@ -113,6 +118,28 @@ def benchmark(self, mode: str = "train", epoch: int = 0, num_steps: int = 1000,
                 print("FastEstimator: Step: {}, Epoch: {}, Steps/sec: {}".format(idx, epoch, iters_per_sec))
                 start = time.perf_counter()
 
+    def transform(self, data: Dict[str, Any], mode: str, epoch: int = 0):
```

Comment:
what if user puts in batched data? like if they want to inference 4 images at once

Reply:
they will have to call infer 4 times.    the infer is designed to work one data at a time.

Reply:
suppose a user has a model which needs 2 images of horse and 2 images of zebra in order to forward pass

Reply:
the 2images of horse and zebra won't be appearing on batch dimension. they will be appearing on different keys like "horse1" "horse2"...